include/dogecoin/qrengine.h

```diff
@@ -0,0 +1,26 @@
+// QR Engine using Project Nanuki QR (MIT) and lodepng (ZLIB) for encoding.
+
+// (c) 2023 michilumin
+// (c) 2023 The Dogecoin Foundation
+
+#ifndef __LIBDOGECOIN_QRENGINE_H__
+#define __LIBDOGECOIN_QRENGINE_H__
+
+#include <dogecoin/cstr.h>
+#include <dogecoin/dogecoin.h>
+
+// internal funcs not exported
+void printQr(const uint8_t qrcode[]);
+int outputQRStringFromQRBytes(const uint8_t* inQrBytes, char* outString);
+int stringToQrArray(const char* inString, uint8_t* outQrBytes);
```

Comment:
A couple inconsistent parameter names and formatting between declaration and definition

qrengine.h:14 vs qrengine.c:35
```
int outputQRStringFromQRBytes(const uint8_t* inQrBytes, char* outString);
int outputQRStringFromQRBytes(const uint8_t qrcode[],char* outstring)
```

qrengine.h vs qrengine.c:18
```
int stringToQrArray(const char* inString, uint8_t* outQrBytes);
int stringToQrArray(const char* inString,uint8_t* qrcode)
```

Reply:
Fixed, normalized to what was in qrengine.h